firebase/functions/src/index.ts

```diff
@@ -179,6 +205,21 @@ export const createChain = functions
           chainId: chainData.id,
           role: user.customClaims?.role ?? ROLE_CHAINADMIN,
         });
+
+        let name;
+        let email;
+        try {
+          name = user.displayName!;
+          email = user.email!;
+
+          addContactToMailchimpAudience(name, email);
```

Comment:
Shouldn't we have a checkmark if the user actually wants to subscribe? Always subscribing is probably not a good idea. Also, don't forget to await here.